starcheck/src/starcheck.pl

```diff
@@ -44,13 +42,18 @@
 use HTML::TableExtract;
 
 use Ska::AGASC;
+use Carp 'verbose';
+$SIG{ __DIE__ } = sub { Carp::confess( @_ )};
```

Comment:
If I raise an error on purpose in, say, the main plotting routine, here's what the old output was:

```
exceptions.ValueError: Raise error in Python plotting to show output at line 19
```
the new output looks like:

```
exceptions.ValueError: Raise error in Python plotting to show output at line 19
 at ./starcheck/src/starcheck.pl line 46
        main::__ANON__('exceptions.ValueError: Raise error in Python plotting to show...') called at (eval 109) line 3
        main::plot_cat_wrapper('__main__', 'plot_cat_wrapper', 'HASH(0x575db10)') called at ./starcheck/src/starcheck.pl line 616
```
It isn't perfect, because you still can't see that the line that raised the unhandled exception was in starcheck.plot, but at least it is isolated to the plot_wrapper.  There should be a better way to properly control the Python traceback printing or all exception printing  (to make it more normal), but I haven't figured it out.